lib/cli.rb

```diff
@@ -83,6 +83,9 @@ def self.handle_error(e)
     Machinery::Ui.kill_pager
 
     case e
+    when OptionParser::AmbiguousOption
```

Comment:
Andre is right, it should be showing up so you can find out what those ambiguous options are ... sorry that I didn't think of it :(